cypress/pageobjects/template.po.ts

```diff
@@ -23,19 +26,24 @@ export default class TemplatePage extends CruResourcePo {
     });
   }
 
-  public setValue(value: ValueInterface) {
-    this.namespace().select({option: value?.namespace})
-    this.name().input(value?.name)
-    this.description().input(value?.description)
-    this.cpu().input(value?.cpu)
-    this.memory().input(value?.memory)
+  public setBasics(cpu?: string, memory?: string) {
+    this.clickTab('Basics');
```

Comment:
I prefer do not inherit the function `setBasics` of VMs Class since different tab id, one of tabs is `basics`, the other one is `Basics`.